WCMST1.sql_schema

```diff
@@ -153,8 +153,15 @@ CREATE TABLE IF NOT EXISTS `users` (
   UNIQUE KEY `username` (`username`)
 ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci AUTO_INCREMENT=10 ;
 
+--
+-- Insert username- 'admin' and MD5 has of the password 'admin-password'
+--
+INSERT INTO `users`(`username`, `password`) VALUES ('admin','4b99e4ad8eabc68770e4c2bdf32298d8');
 
 
 /*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
 /*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
 /*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
+
+
+-- DROP TABLESPACE `achievements`,`events`,`faculty`,`gallery`,`news`,`notifications`,`papers`,`users`;
```

Comment:
A bit confused about what is to be done here. `DROP TABLESPACE` was added to fix a data persistence issue on the MySQL container, which I believe is no longer required as of now.

Is am not sure about the `DROP TABLE [IF EXISTS] [database_name.][schema_name.]table_name;` you suggested.

Reply:
I think it would be better for you to fix this in a seperate issue/PR.

Reply:
> Hi @sebinbenjamin , An issue for this #1 exists and a pull request #12 has been submitted. However, we still require following extra additions of this commit you have done in the SQL dump file.
> 
> 1. Change of DB name from 'ce'  to wcms.
> 2. Removing the 'achievements' table from schema and if exists already in db line 167. Use
>    `DROP TABLE [IF EXISTS]  [database_name.][schema_name.]table_name;` . Using IF EXISTS will help backward support for those who have already created the DB.
> 3. Other changes that you have done in .gitignore file and readme of port is accepted in #9  pull request
> 
> You may raise a single new issue for resolving 1 and 2 above. Either you may take it up or I will do it. Ping back based on your availability for resolving it.

Raised Issue #14